osd_install_access_delete_cluster/creating-an-aws-cluster.adoc

```diff
@@ -27,3 +27,4 @@ include::modules/osd-create-cluster-red-hat-account.adoc[leveloffset=+1]
 * For information about load balancers for {product-title}, see the xref:../osd_architecture/osd_policy/osd-service-definition.adoc#load-balancers_osd-service-definition[Load balancers] section in the {product-title} service definition.
 * For more information about etcd encryption, see the xref:../osd_architecture/osd_policy/osd-service-definition.adoc#etcd-encryption_osd-service-definition[etcd encryption service definition].
 * For information about the end-of-life dates for {product-title} versions, see the xref:../osd_architecture/osd_policy/osd-life-cycle.adoc#osd-life-cycle[{product-title} update life cycle].
+* For information about the requirements for custom additional security groups, see xref:../osd_planning/aws-ccs.adoc#osd-security-groups-custom_aws-ccs[Security groups].
```

Comment:
Because you're linking to the "Additional custom security groups" section, consider changing the link text here to match that heading.

Reply:
changed